news/flake8-2.rst

```diff
@@ -0,0 +1,25 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* Added flake8 builtins exception for ``__xonsh__``, 
+  should not need ``import builtins``, ``builtins.__xonsh__`` any more
```

Comment:
I actually like the importing of builtins. Is this a problem for some reason?

Reply:
`import builtins` struck me as unpythonic, builtins are supposed to just be there.  But I felt any explicit import ought to be named something xonsh-ish.  I kept going back and forth on this but settled for the simple flake8 config change and no import.  That seems appropriate for what should be a superficial lint fix PR.

But we're talking about the xonsh public API and there remain things that nag at me: why the xonsh session is called `__xonsh__` (which seemingly "reserved" symbol is nevertheless freely referenced by most code modules; why a rarely referenced attribute like `execx` is given the builtin treatment but a frequently referenced attribute like `__xonsh__.env` is not; and whether there is a performance benefit to the 'dynamic proxy' (== xonsh builtin) to justify the special treatment? I'd like to hash these out some day and formalize the xonsh API,, but on some other PR.

Reply:
Oh that is funny.  I thought `import builtins` is pythonic because it makes it explicit where these variables live

Reply:
Caught me! I am no arbiter of what's pythonic and should not be throwing shade about it! But [python builtins](https://docs.python.org/3/library/builtins.html) does say explicitly that it is not normally imported and the one example it provides for when it might be is nothing like this.
In this linting pass, I found most modules were not importing builtins before reference, but some were, aided and abetted by Flake8 per-module exceptions..  How should we make this consistent across the project (and maybe doc in guide)?  Conciseness led me to the current PR.  Or we could add the import in all the necessary places.  Please advise your preference.
Ultimately, (in some future PR), I would like to finally break the (confusing, IMHO) association of Xonsh session externals and the python `builtins` module -- a subject for discussion another day.